_posts/grant_guo/2018-03-22-kf-etl.md

```diff
@@ -0,0 +1,349 @@
+---
+layout: post
+title:  "Introducing the Kids-First ETL"
+breadcrumb: true
+author: grant_guo
+date: 2018-03-19
+categories: grant_guo
+tags:
+    - Functional Programming(FP), Scala
+teaser:
+    info: An introduction of how to apply Scala's functional programming features to the Kids-First ETL
+    image: lambda.png
+header:
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+## Abstractions
```

Comment:
Can you add a very brief introduction on what you'll be presenting ? A bit of context. The idea is for someone without this particular expertise to understand what the blog post is about.